src/Microsoft.ML.Transforms/OneHotHashEncoding.cs

```diff
@@ -189,7 +189,7 @@ internal OneHotHashEncodingTransformer(HashingEstimator hash, IEstimator<ITransf
         /// </summary>
         public IDataView Transform(IDataView input) => _transformer.Transform(input);
 
-        public void Save(ModelSaveContext ctx) => _transformer.Save(ctx);
+        void ICanSaveModel.Save(ModelSaveContext ctx) => (_transformer as ICanSaveModel).Save(ctx);
```

Comment:
>as ICanSaveModel [](start = 71, length = 16)

Similar about `()` style casts.

Reply:
@tfinley@gmail.com, do you mean we need to use `(ICanSaveModel)_transformer` because `_transformer` must implement `ICanSaveModel`?

---
In reply to: [254489139](https://github.com/dotnet/machinelearning/pull/2431#discussion_r254489139) [](ancestors = 254489139)